### PR TITLE
dnsmasq: more config fixes

### PR DESCRIPTION
**Pull request by aermakov-zalando**
* always try to use local CoreDNS first
 * log to stdout

### CODE REVIEW

File: cluster/manifests/coredns-local/daemonset-coredns.yaml [hunk 1]
```diff
@@ -55,11 +55,13 @@ spec:
...
         - --keep-in-foreground
+        - --log-facility=-
         - --cache-size=50000
-        - --dns-forward-max=100
+        - --dns-forward-max=500
         - --neg-ttl=60
-        - --server=127.0.0.1#9254
+        - --strict-order
         - --server=10.3.0.11#53
+        - --server=127.0.0.1#9254
```
Comment (mikkeloscar) [inline cluster/manifests/coredns-local/daemonset-coredns.yaml:65]: Maybe add a comment on the order of `--server` to make it very clear for the future!!

Comment (aermakov-zalando) [reply cluster/manifests/coredns-local/daemonset-coredns.yaml:65]: Good point, added!

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**